tools/python/boutiques/validator.py

```diff
@@ -58,6 +62,82 @@ def inById(i):
             return descriptor["inputs"][inputGet("id").index(i)]
         return {}
 
+    def conditionalExpFormat(s):
```

Comment:
This function appears to exist in both `localExec.py` and `validator.py`, so perhaps we could move it to a `utils` file and import it?

Reply:
Function moved to utils.py!